pkg/console/controllers/route/controller.go

```diff
@@ -159,6 +163,124 @@ func (c *RouteSyncController) Sync(ctx context.Context, controllerContext factor
 	return statusHandler.FlushAndReturn(defaultRouteErr)
 }
 
+func (c *RouteSyncController) updateIngressConfigStatus(ctx context.Context, routeConfig *routesub.RouteConfig, customRouteErrors []error) error {
+	componentRoute := &configv1.ComponentRouteStatus{
+		Namespace:        api.OpenShiftConsoleNamespace,
+		Name:             c.routeName,
+		DefaultHostname:  configv1.Hostname(routeConfig.GetDefaultRouteHostname()),
+		CurrentHostnames: []configv1.Hostname{configv1.Hostname(routeConfig.GetCustomRouteHostname())},
```

Comment:
Seems like current hostname could be set to an empty string if [this defer statement](https://github.com/openshift/console-operator/pull/553/files#diff-af15b762c545564ccd44f8bf5c8592e4233e4836241f99b05bd73f8109c181daR323) is executed after the custom route fails validation.